spec/helpers/leaderboard_helper_spec.rb

```diff
@@ -0,0 +1,37 @@
+RSpec.describe LeaderboardHelper, type: :helper do
+  describe "#participation_points_active?" do
+    context "when points are awarded for participation" do
+      before { create(:rule, key: :points_for_participation, value: 1) }
+
+      it { expect(helper.participation_points_active?).to eq(true) }
+    end
+
+    context "when points are not awarded for participation" do
+      it { expect(helper.participation_points_active?).to eq(false) }
+    end
```

Comment:
Seems more natural as-is to me. I don't have any grounds to object to the change, but I'll leave them as they are.